themes/default/content/registry/packages/aws/installation-configuration.md

```diff
@@ -293,6 +293,17 @@ Make sure that your local environment does not have AWS credentials configured b
 
 To learn more about projecting environment variables in Pulumi ESC, refer to the [relevant Pulumi ESC documentation](/docs/pulumi-cloud/esc/environments/#projecting-environment-variables).
 
+
+### Authenticate with WebIdentity and OpenID Connect (OIDC)
+
+In this approach, you configure an AWS role to assume and a source for a web identity token, which is an OIDC ID token. The token is used to authenticate with AWS STS (Security Token Service) and obtain temporary credentials. The temporary credentials are then used to authenticate with AWS. Please refer to the AWS docs [Assume role with web identity](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-role.html#cli-configure-role-oidc) for more details.
+
+This mode of authentication allows you to run Pulumi on a service that supports OIDC like GitHub, GitLab, or Azure DevOps, and access AWS resources without storing credentials.
+Refer to the particular service's documentation for how to configure the OIDC trust relationship, which is a one-time setup.
+
+For this provider, configure the `assumeRoleWithWebIdentity` object documented below, where you need at least `roleArn` for the role to assume and one of `webIdentityToken` or `webIdentityTokenFile` for the ID token.
```

Comment:
I recommend including an example of the config file contents so they can see what it should look like:

```yaml
config:
  aws:assumeRoleWithWebIdentity:
    roleArn: arn:aws:iam::058111598222:role/OrganizationAccountAccessRole
    // Define either webIdentityToken or webIdentityTokenFile
    webIdentityToken: <your-web-identity-token>
    webIdentityTokenFile: webidentitytokenfile.txt
```